WTG.Analyzers.Test/TestData/DiscardLocalAnalyzer/Deconstruct.Source.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+
+public abstract class Bob
+{
+	public void M1()
+	{
+		var (x, _) = N1();
+	}
+
+	public void M2()
+	{
+		(var x, _) = N1();
+	}
+
+	public void M3()
+	{
+		(var x, var _) = N1(); // '_' is still treated as a discard here, even though 'var' makes it look like a local.
```

Comment:
Interesting!

Reply:
I suspect it was an oversight. having `(var x, _) = N1()` and `(var x, var _) = N1()` both mean the same thing doesn't feel like an intentional design choice. I would expect that either the latter defines a variable or one of them produces a warning/error.

Reply:
I can't find anything in the spec for this, raised https://github.com/dotnet/csharplang/issues/3229 as a question.